settings.gradle

```diff
@@ -5,21 +5,17 @@ buildscript {
         }
     }
     dependencies {
-        classpath "gradle.plugin.ch.myniva.gradle:s3-build-cache:0.9.0"
+        classpath "gradle.plugin.ch.myniva.gradle:s3-build-cache:0.10.0"
         classpath "com.gradle.enterprise:com.gradle.enterprise.gradle.plugin:3.2.1"
     }
-
-    configurations.classpath {
-        resolutionStrategy {
-            force 'com.amazonaws:aws-java-sdk-s3:1.11.582'
-        }
-    }
 }
 
 apply plugin: 'ch.myniva.s3-build-cache'
 apply plugin: 'com.gradle.enterprise'
 
-ext.isMasterBuild = (System.getenv("CIRCLE_BRANCH") ?: System.getenv("BUILD_SOURCEBRANCHNAME")) == "master"
+ext.isMasterBuild = false ||
```

Comment:
Do we have to add the false there? Wouldn't Groovy truth validate correctly otherwise as well?

Reply:
this is to align the next lines :)